app/Metadata/Geodecoder.php

```diff
@@ -34,7 +34,7 @@ public static function getGeocoderProvider(): ProviderCache
 
 			$httpClient = new \GuzzleHttp\Client([
 				'handler' => $stack,
-				'timeout' => Configs::get_value('location_decoding_timeout'),
+				'timeout' => Configs::getValueAsString('location_decoding_timeout'),
```

Comment:
```suggestion
				'timeout' => Configs::getValueAsInt('location_decoding_timeout'),
```

Have a look into the Guzzle docs or into `\GuzzleHttp\Handler\CurlFactory`. The value is used as an integer.